pyproject.toml

```diff
@@ -21,8 +21,8 @@ enum-tools = { extras = ["sphinx"], version = "0.9.0.post1", optional = true }
 crypto-cpp-py = "^1.0.4"
 marshmallow = "^3.15.0"
 marshmallow-oneofschema = "^3.0.1"
-cairo-lang = "0.10.2"
 typing-extensions = "^4.3.0"
+cairo-lang = "==0.10.3"
```

Comment:
I'm not sure if `==` is required, or it can just be `0.10.3`. But either is probably fine.

Reply:
https://python-poetry.org/docs/dependency-specification/#exact-requirements